prow/cmd/deck/static/plugin-help-script.js

```diff
@@ -4,215 +4,366 @@ const closedArrow = "\u25b6 ";
 const openedArrow = "\u25bc ";
```

Comment:
Might be nice to get your formatter to follow current style or do formatting pass as a separate commit/diff

Reply:
What is our current style? I will config my formatter to follow the style. Currently, I am using 2 spaces and wrap at 80.

Reply:
I would just compare what the commit before had ... looks like four spaces for indentation and no line width max that I can tell at all.

Reply:
Fixed indent and wrap